news/avoid_setup_package_uninstall.rst

```diff
@@ -0,0 +1,25 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* Build templates only install dependencies of ``conda-forge-ci-setup`` package if ``local_ci_setup``, removing the need to uninstall it forcefully,
+  which does not behave as documented https://github.com/conda/conda/issues/10148
```

Comment:
I don't think we need to worry about the behavior of conda changing here. The docstring is clearly wrong and simply needs to be fixed.

Reply:
Sure the docstring is wrong, but I'm not sure I'd rely on behaviour of a command which (as I've shown on the ticket) would attempt to uninstall curl if one uninstalls git

Reply:
We're using --force.